perma_web/perma/templates/user_management/settings-affiliations.html

```diff
@@ -0,0 +1,86 @@
+{% extends "settings-layout.html" %}
+
+{% block title %} | Settings | Your Affiliations{% endblock %}
+
+{% block dashboardContent %}
+    <h3 class="body-ah _hideMobile">Your Affiliations</h3>
+    {% if messages %}
+    {% for message in messages %}
+      <div class="alert-{{ message.level_tag }} alert-block">{% if 'safe' in message.tags %}{{ message|safe }}{% else %}{{ message }}{% endif %}</div>
+    {% endfor %}
+    {% endif %}
+    <div class="org-settings">
+      {% for registrar, orgs in orgs_by_registrar.items %}
+        <h4 class="body-bh">
+          <a href="mailto:{{registrar.email}}">{{registrar.name}}</a>
+        </h4>
+        {% for org in orgs %}
+          <div class="settings-block">
+            <p>
+              {{ org }}
+              <span class="_hideMobile">
+                {% if org.default_to_private %}
+                  (Links are <strong>Private</strong> by default)
+                {% else %}
+                  (Links are <strong>Public</strong> by default)
```

Comment:
Small suggestion -- let's not include `(Links are Public by default)`, only Private, because most users will *only* have public orgs and never think about it.